bifrost.go

```diff
@@ -108,3 +109,15 @@ func newGoogleCloudStorage(g *BridgeConfig) (rainbowBridge, error) {
 		PublicRead:      g.PublicRead,
 	}, nil
 }
+
+// newGoogleCloudStorage returns a new client for AWS S3
```

Comment:
I think the comment has a name conflict. It should not be `newGoogleCloudStorage`